tests/socket/socket_tests.cpp

```diff
@@ -269,7 +269,10 @@ connection_monitor_test(
 
     // Ring buffer event callback context.
     std::unique_ptr<ring_buffer_test_event_context_t> context = std::make_unique<ring_buffer_test_event_context_t>();
-    context->test_event_count = disconnect ? 4 : 2;
```

Comment:
what about the ports on the expected records for connection_established event?

Reply:
Tracked by #2706